js/cypress/support/generatePassword.ts

```diff
@@ -0,0 +1,13 @@
+import { webcrypto as crypto } from "crypto";
+
+export function generatePassword(length: number): string {
+  const lowestPrintableCode = 33;
+  const highestPrintableCode = 126 - lowestPrintableCode;
```

Comment:
If I'm understanding correctly, this variable name is misleading, as 126 is the highest printable ASCII code, and this variable is the difference between the highest code and the lowest. If this was the highest printable code, the arithmetic at line 10 would produce non-printable codes.